src/components/templates/ScreeningRoom/ScreeningRoom.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useMemo, useCallback, useState } from "react";
+import { faSearch, faTimesCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { IFRAME_ALLOW } from "settings";
+
+import { GenericVenue } from "types/venues";
+import { ScreeningRoomVideo } from "types/screeningRoom";
+
+import { WithId } from "utils/id";
+
+import { useScreeningRoom } from "./useScreeningRoom";
```

Comment:
Should this go to the bottom of the imports since its in the same dir?

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1420/commits/a548b05be944d45a37282dc7de1064038aa08f7f

Reply:
> Should this go to the bottom of the imports since its in the same dir

@denisdimitrov @mike-lvov Yup. Though still in the same vague 'hierarchy' within that 'local imports' (`./`) section (so hooks, then components, then scss, etc)

(the change @mike-lvov made is correct, I just wanted to be clear about the answer so you know for sure in future)